addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -24,3 +24,30 @@ table.o_section_list_view tr.o_data_row.o_is_section {
 .icon_rotates {
     transform: rotate(180deg);
 }
+
+.o_survey_sample_tile {
```

Comment:
Would add a small header + docstring, explaining a bit its purpose (easier when reading the scss file)